vita3k/gui/src/settings_dialog.cpp

```diff
@@ -250,15 +250,19 @@ void init_config(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path
     }
 
     list_user_lang.clear();
-    const auto LANG_PATH{ emuenv.shared_path / "lang/user" };
-    if (fs::exists(LANG_PATH) && !fs::is_empty(LANG_PATH)) {
-        for (const auto &lang : fs::directory_iterator(LANG_PATH)) {
+    const auto get_list_user_lang = [&](const fs::path &path) {
+        for (const auto &lang : fs::directory_iterator(path / "lang/user")) {
             if (lang.path().extension() == ".xml") {
                 const auto lang_file_name = lang.path().filename().replace_extension().string();
                 list_user_lang.push_back(lang_file_name);
             }
         }
-    }
+    };
+
+    get_list_user_lang(emuenv.static_assets_path);
+#ifndef WIN32
+    get_list_user_lang(emuenv.shared_path);
+#endif
```

Comment:
Wouldn't this cause a problem (duplicate entry) if emuenv.static_assets_path = emuenv.shared_path (which can also happen on a non-Windows OS) ?

Reply:
Only Linux have it, and Android so for Mac  🤔

Reply:
i have changed it little, normally should fine for all case